agent/runner/actions/mongodb_explain_action_test.go

```diff
@@ -304,6 +304,9 @@ func TestMongoDBExplain(t *testing.T) {
 			want["optimizationTimeMillis"] = map[string]interface{}{"$numberInt": "0"}
 			want["winningPlan"] = map[string]interface{}{"stage": "EOF", "isCached": false}
 			want["prunedSimilarIndexes"] = false
+			if mongoDBVersion.Minor >= 2 {
```

Comment:
@idoqo If this is applicable to 8.2+, then we should probably check the major version as well. Otherwise it may fail for 7.2, for example.